### PR TITLE
Kamu UI 139 remove view encapsulation.none and refactor styles

### PR DESCRIPTION
**Pull request by dmitriy-borzenko**

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
 @import "bootstrap-icons/font/bootstrap-icons.css";
 @import "bootstrap/scss/bootstrap";
 @import "@danielmoncada/angular-datetime-picker/assets/style/picker.min.css";
+@import "../src/assets/styles/components/index";
```
Comment (EvilAlexei) [inline src/styles.scss:6]: Previously I added a path alias for `var.scss`, and it should work for the index too.
```suggestion
@import "components/index";
```

File: src/app/dataset-view/dataset.component.scss [hunk 2]
```diff
@@ -1,145 +1,149 @@
...
         }
 
-        .datatab-activity-timestamp {
-            color: rgb(0 0 0 / 66%);
-            flex-shrink: 0;
-            text-align: right;
-            width: 62px;
-            margin-right: 10px;
-        }
+        .datatab-expanding-row {
```
Comment (EvilAlexei) [inline src/app/dataset-view/dataset.component.scss:18]: Styles that are from other components should be inside those components (if it's not a 3rd party).
In this case, it should be in the `data.component`

Please check other styles from this file.

File: src/app/dataset-view/dataset.component.scss [hunk 3]
```diff
@@ -1,145 +1,149 @@
...
+            .mat-button-toggle-group-appearance-standard {
+                border-radius: 0 !important;
+            }
         }
 
-        .datatab-activity-row-summary {
-            display: flex;
+        #dataset-view-history {
+            overflow: visible;
         }
```
Comment (EvilAlexei) [inline src/app/dataset-view/dataset.component.scss:16]: These styles should be in the history.component

File: src/app/dataset-view/dataset.component.scss [hunk 4]
```diff
@@ -1,145 +1,149 @@
...
-        line-height: 20px;
-        cursor: pointer;
-        display: flex;
-        padding: 4px 10px;
-
-        &-content {
-            flex-grow: 1;
-            max-width: calc(100% - 24px);
+:host {
+    ::ng-deep {
```
Comment (EvilAlexei) [inline src/app/dataset-view/dataset.component.scss:4]: Inside `:host ::ng-deep` should be only 3rd party styles,
styles from child components should be in their own style files

**Review by EvilAlexei — CHANGES_REQUESTED** [review]

File: src/app/dataset-block/metadata-block/components/tooltip-icon/tooltip-icon.component.scss [hunk 5]
```diff
@@ -1,26 +0,0 @@
-.tooltip-class {
```
Comment (EvilAlexei) [inline src/app/dataset-block/metadata-block/components/tooltip-icon/tooltip-icon.component.scss:1]: Please remove this file
And don't forget to run stylelint along with lint  :)

I have plans to add a pre-commit hook with lint + stylelint, for safe commits :D

**Review by EvilAlexei — CHANGES_REQUESTED** [review]

File: src/assets/styles/components/_tooltip.scss [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    margin-bottom: 5px;
+}
+
+.bs-tooltip-top,
+.bs-tooltip-bottom {
+    .tooltip-arrow {
+        &::before {
+            border-top-color: #ced4da !important;
+            border-bottom-color: #ced4da !important;
+            top: -5px !important;
```
Comment (EvilAlexei) [inline src/assets/styles/components/_tooltip.scss:23]: I think it should work without `!importants` - I checked a few places, and it was OK.
But I could miss something. If there are any style collisions - please contact me, and we could try to solve them together

File: src/assets/styles/components/_tooltip.scss [hunk 7]
```diff
@@ -0,0 +1,30 @@
+.tooltip-class {
+    z-index: 1000;
+}
+
+.tooltip-inner {
+    background-color: #f3f4f6;
+    position: relative;
+    font-size: 12px;
+    border: 1px solid #ced4da;
```
Comment (EvilAlexei) [inline src/assets/styles/components/_tooltip.scss:9]: I suggest adding a variable for border color,
there are three places here with this color, and I think it should be the same all the time.
For example `$border-color: #ced4da;` and replace border-color with this variable in all 3 places

**Review by EvilAlexei — CHANGES_REQUESTED** [review]

File: src/styles.scss (hunk 1 above)
Comment (dmitriy-borzenko) [reply src/styles.scss:6]: Fixed

File: src/app/dataset-view/dataset.component.scss (hunk 4 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/dataset.component.scss:4]: Fixed

File: src/app/dataset-block/metadata-block/components/tooltip-icon/tooltip-icon.component.scss (hunk 5 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-block/metadata-block/components/tooltip-icon/tooltip-icon.component.scss:1]: Ok, removed

File: src/assets/styles/components/_tooltip.scss (hunk 6 above)
Comment (dmitriy-borzenko) [reply src/assets/styles/components/_tooltip.scss:23]: Yes, thanks. Removed

File: src/assets/styles/components/_tooltip.scss (hunk 7 above)
Comment (dmitriy-borzenko) [reply src/assets/styles/components/_tooltip.scss:9]: Ok. fixed

File: src/app/dataset-view/dataset.component.scss (hunk 2 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/dataset.component.scss:18]: Moved to separate file

File: src/app/dataset-view/dataset.component.scss (hunk 3 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/dataset.component.scss]: Moved to history.component

**Review by EvilAlexei — APPROVED** [review]

**Pull request merged**